static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/control.js

```diff
@@ -1,41 +1,38 @@
 // @flow
-import type { CurrentABTest, SignInGateVariant } from '../types';
+import type { SignInGateVariant } from '../types';
 import { componentName } from '../component';
 import {
     hasUserDismissedGate,
     isNPageOrHigherPageView,
     isLoggedIn,
     isInvalidArticleType,
     isInvalidSection,
+    isIOS9,
 } from '../helper';
 
 // pull in the show method from the design folder, which has the html template and and click handlers etc.
-import { designShow } from './design/quartus';
+// No design needed as not showing a sign in gate in the control
 
 // define the variant name here
 const variant = 'control';
 
 // method which returns a boolean determining if this variant can be shown on the current pageview
 const canShow: (name?: string) => boolean = (name = '') =>
     !hasUserDismissedGate({
-        componentName,
         name,
         variant,
+        componentName,
     }) &&
-    isNPageOrHigherPageView(2) &&
+    isNPageOrHigherPageView(3) &&
```

Comment:
we're showing both control and variant on the 2nd or higher pageview now, so this should be `2`